app/models/post.rb

```diff
@@ -1,16 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user, foreign_key: 'user_id'
   has_many :comments
   has_many :likes
 
-  # Updates the posts counter for a user
-  private
+  validates :title, presence: true, length: { minimum: 3, maximum: 250 }
+  validates :comment_counter, numericality: { greater_than_or_equal_to: 0, only_integer: true }
+  validates :likes_counter, numericality: { greater_than_or_equal_to: 0, only_integer: true }
+  after_save :post_count_updater
 
-  def update_user_posts_counter
-    author.update(posts_counter: author.posts.count)
+  def post_count_updater
+    user.update(posts_counter: user.posts.count)
   end
```

Comment:
- [ ] Great work so far but i guess you misunderstood the requested changes because previous reviewer requested you to test these updater method for posts_counter, likes_counter and comments_counter 🤔 I can see that you are testing the columns and these methods are being tested implicitly but it would be nice if you manually run these methods in tests file to test them explicitly 👍